code/__DEFINES/mobs.dm

```diff
@@ -196,6 +196,29 @@
 #define INVISIBILITY_ABSTRACT 101
 #define UNHEALING_EAR_DAMAGE 100
 
+/* Species organ defines */
+// Internal Organs (has_organ)
+#define ORGAN_HEART	   "heart"
+#define ORGAN_LUNGS	   "lungs"
+#define ORGAN_LIVER	   "liver"
+#define ORGAN_KIDNEYS  "kidneys"
+#define ORGAN_BRAIN	   "brain"
+#define ORGAN_APPENDIX "appendix"
+#define ORGAN_EYES	   "eyes"
+
+// External Organs (has_limbs)
+#define ORGAN_CHEST  "chest"
+#define ORGAN_GROIN  "groin"
+#define ORGAN_HEAD	 "head"
+#define ORGAN_ARM_L  "l_arm"
+#define ORGAN_ARM_R  "r_arm"
+#define ORGAN_LEG_L  "l_leg"
+#define ORGAN_LEG_R  "r_leg"
+#define ORGAN_HAND_L "l_hand"
+#define ORGAN_HAND_R "r_hand"
+#define ORGAN_FOOT_L "l_foot"
+#define ORGAN_FOOT_R "r_foot"
```

Comment:
Potentially a future problem creator.

Reply:
Sorry, could you clarify what the problem is here? I'm not sure I understand.

Reply:
I'll put this here too so it doesn't get lost to time:
![image](https://user-images.githubusercontent.com/57483089/120160800-902a4a80-c1ee-11eb-947a-6dffee579c42.png)